kernel/main.rs

```diff
@@ -136,6 +136,10 @@ impl KernelOps for ApiOps {
         register_ethernet_driver(driver)
     }
 
+    fn register_block_driver(&self, driver: Box<dyn kerla_api::driver::block::BlockDriver>) {
+        register_block_driver(driver)
```

Comment:
This causes an infinite recursion. I think this is the root cause of the kernel panic.

Reply:
Since Kerla still doesn't have the block I/O layer, let's leave this method empty.

Reply:
Thank you! Would've took a while to catch that. All that's seems to be left now is interrupt handling and making a proper read config method that reads the block size properly.